.clang-format

```diff
@@ -80,10 +80,11 @@ IncludeCategories:
 IncludeIsMainRegex: '(Test)?$'
 IncludeIsMainSourceRegex: ''
 IndentCaseLabels: false
-IndentGotoLabels: true
+IndentGotoLabels: false
```

Comment:
goto labels are important to be able to quickly identify. Having them left-aligned in the text helps provide that visual indication.